### DISCUSSION TITLE
Editable Cells in the Table

### DISCUSSION THREAD

**Discussion by Callumk7** [adobe/react-spectrum#6382] [Q&A] [answered]
Hey all, 

What is the recommended way to allow a user to edit a cell in a table? I am finding any input fields behave strangely due to the keyboard navigation defaults of the table component, which as far as I can see, I cannot disable. Thanks in advance for the advice.

**Comment by reidbarber** [comment, marked as answer]
We have an issue for this here: https://github.com/adobe/react-spectrum/issues/2328

One alternative for now is to open a dialog from the cell and allow the field to be edited there.

**Comment by Callumk7** [comment]
Nice, thanks for the swift response - I will keep an eye out on that issue 👍
